File: scratch/inspect_ledger.js
const fs = require('fs');
const path = require('path');

const filePath = path.join(__dirname, '../client/src/admin/CustomerLedger.jsx');
const content = fs.readFileSync(filePath, 'utf8');

console.log('Includes duplicateName hook:', content.includes('duplicateName'));
console.log('Includes handleDuplicateCustomer:', content.includes('handleDuplicateCustomer'));
console.log('Includes Duplicate button:', content.includes('👯 Duplicate'));
console.log('Includes isDuplicateModalOpen markup:', content.includes('Duplicate {profile.ledgerType}'));

File: scratch/inspect_ledger.py
file_path = r"d:\KSK\HOST\KSK REACT\KSK1\V_1 - Main\client\src\admin\CustomerLedger.jsx"

with open(file_path, "r", encoding="utf-8") as f:
    content = f.read()

import re

for match in re.finditer("duplicateMobile", content):
    start = match.start()
    print(f"duplicateMobile found at {start}:")
    print(repr(content[start-100:start+250]))

for match in re.finditer("editMobile", content):
    start = match.start()
    print(f"editMobile found at {start}:")
    print(repr(content[start-100:start+250]))
